base/essentials.jl

```diff
@@ -158,6 +158,11 @@ macro boundscheck(blk)
       $(Expr(:boundscheck,:pop)))
 end
 
+"""
+    @inbounds(blk)
+
+Tells the compiler to skip bound checks within the given block.
```

Comment:
I was planning to replace the above line with this

In the example below the bound check of array A is skipped to improve performance

```
function sum(A::AbstractArray)
    r = zero(eltype(A))
    for i = 1:length(A)
        @inbounds r += A[i]
    end
    return r
end
```
Or should I add the following link http://docs.julialang.org/en/latest/devdocs/boundscheck